src/src/AdminBundle/Controller/EventController.php

```diff
@@ -36,32 +41,49 @@ public function createAction(Request $request)
     }
 
     /**
-     * @Route("/event/edit/{id}", name="event_edit")
+     * @Route("event_type/edit/{id}/event/edit/{event_id}", name="event_edit")
      * @Method({"GET", "POST"})
      */
-    public function editAction()
+    public function editAction($id, $event_id, Request $request)
     {
-        return $this->render('AdminBundle:Event:index.html.twig');
+        $repository = $this->getDoctrine()->getRepository(Event::class);
+        $event = $repository->findOneBy(['id' => intval($event_id)]);
+
+        $form = $this->createForm(EventForm::class, $event);
+
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            $em = $this->getDoctrine()->getManager();
+            $em->persist($event);
```

Comment:
tento riadok mi tu pride zbytocny, teda v ziadej inej `editAction` nepouzivam `persist` a funguje to.

Reply:
mozno to funguje len preto, lebo je to debilproof 😁 jk

Reply:
No rada by som to tam nechala.

Reply:
nikde inde to nieje, dajme to prec aj tu.